package.json

```diff
@@ -50,6 +50,7 @@
 	],
 	"devDependencies": {
 		"@sindresorhus/tsconfig": "^1.0.1",
+		"@type-challenges/utils": "^0.1.1",
```

Comment:
is needed for `Expect` and `Equal`

however, I can define it without dependency